src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -98,25 +77,27 @@ String privacySettingButtonValue = (Boolean) request.getAttribute("isPrivate")?
 
   <div id="container">
 
-    <h1>
-      <!-- Conversation title -->
-      <%= conversation.getTitle() %>
-        
-      <!-- Setting button and content -->
-      <div id="dropdown-settings">
-        <i id="conversationSettings" class="fa fa-cog"> </i>
-        <div class="dropdown-content" style="display:none">
-          <li> <button id="privacySettingButton"> <%=privacySettingButtonValue%> </button> </li>
-          <li> <button id="addUsersModalTrigger"> Add User </button> </li>
+    <div class="headerContainer">
+
+      <div class="titleAndSettings">
+        <!-- Conversation title -->
+        <h1> <%= conversation.getTitle() %> </h1>
+          
+        <!-- Setting button and content -->
+        <div class="dropdown">
+          <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> <i class="fa fa-cog"> </i>
```

Comment:
Oh I was afraid changing that class name would mess with the bootstrap css but it didn't. I'm a bit surprised it didn't though.

Was this what you had in mind for the long lines?
<img width="355" alt="screen shot 2018-07-06 at 10 51 14 pm" src="https://user-images.githubusercontent.com/25010539/42407349-277ec816-816f-11e8-9b9b-876929fbc708.png">

Reply:
I was referring to the `id` attribute, not the `class`. Classes can sound general, since they're essentially "types" of elements, but `id`s can't, since they identify exactly one element. So class can still be "dropdown".

Reply:
Typically HTML is wrapped around starting at an attribute, something like:
```
<button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenu2" 
        data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
  <i class="fa fa-cog" />
  ...
```

Reply:
Oh I see, thanks!